masterlist.yaml

```
@@ -3732,13 +3735,36 @@ plugins:
 
 ###### Sim Settlements 2 ######
   - name: 'SS2.esm'
-    url: [ 'https://www.nexusmods.com/fallout4/mods/47976' ]
+    url:
+      - 'https://www.nexusmods.com/fallout4/mods/47976'
+      - 'https://bethesda.net/en/mods/fallout4/mod-detail/4183663'
```

Trailing slashes should be added to URLs where possible for consistency & convenience. Also, it'd be good to add names so that they can be used:

```suggestion
      - link: 'https://www.nexusmods.com/fallout4/mods/47976/'
        name: 'Sim Settlements 2 (Nexus Mods)'
      - link: 'https://bethesda.net/en/mods/fallout4/mod-detail/4183663/'
        name: 'Sim Settlements 2 (Bethesda.net Mods)'
```

---

Again, not sure of the conventions/practices...